tests/test_experiments.py

```diff
@@ -106,6 +107,7 @@ def test_cifar10_cnn_experiment(classification_cfg, tmp_path):
 
 @RunIf(sh=True)
 @pytest.mark.slow
+@pytest.mark.skipif(not _HAS_TIMM, reason="test requires that timm is installed")
```

Comment:
What's the point of this versus the control flow on `CI` environmental variable in `tests/conftest.py`?

Reply:
Good catch! The should have the same control flow. Fixed in b3670713386048c60a36fa44ad949a42a1af0648